webservice-session/session-leon/unit_test/test_todos.py

```diff
@@ -0,0 +1,27 @@
+from src.todos_api import Todos_api
+import pytest
+
+obj = Todos_api
+
+def test_todo_out_of_range_case1():   
```

Comment:
function name can be improved.
The particular case can be specified in the function name.